### DISCUSSION TITLE
Directory.Build.targets, how to avoid CheckNxModuleBoundaries in CI build

### DISCUSSION THREAD

**Discussion by sshelake25** [nx-dotnet/nx-dotnet#908] [Q&A]
Directory.Build.targets, how to avoid CheckNxModuleBoundaries in CI build, 
I have eslint rules for UI projects but no rules for dotnet projects but still its run check module boundies while building dotnet project which has increate solution build time from 2 minutes to 6 minutes.

Local build I is ok, but how I can avoid check module boundies while running building on CI azure pipeline
How to optimized this without impacting build time increse

**Comment by sshelake25** [comment]
What to better manage tags for dotnet mono repo having 100+ projects? Do we have any example repo etc 
Need to enfore module boundries for dotnet projects

**Reply by sshelake25** [reply]
@AgentEnder can you please help on above queries ? its really needed otherwise we will not able to take benefit of nx tagging

**Reply by sshelake25** [reply]
@AgentEnder, how to avoid CheckNxModuleBoundaries command in CI build, if we include it, it incrse build time

**Reply by Tungsten78** [reply]
Concern raised in #841.

As workaround you can remove `BeforeBuild` from `Directory.Build.targets`
